packages/core/src/tracing/sentrySpan.ts

```diff
@@ -401,6 +402,8 @@ export class SentrySpan implements SpanInterface {
       childSpan.spanRecorder.add(childSpan);
     }
 
+    addChildSpanReferenceToSpan(childSpan, this);
```

Comment:
l: nit, but I _think_ I would prefer an order of parent, child, for the arguments 🤔 Makes it a bit easier to reason about IMHO... also the `reference` is a bit redundant here I'd say, what about: 

```js
addChildSpanToSpan(span, childSpan);
```

but no strong feelings, I'm also OK with what is currently here!

Reply:
Sounds good!